package.json

```diff
@@ -35,9 +31,12 @@
     "eslint": "^6.8.0",
     "eslint-config-vtex": "^12.8.1",
     "eslint-config-vtex-react": "^6.7.3",
-    "husky": "^4.2.0",
+    "husky": "^7.0.1",
     "lint-staged": "^10.0.2",
     "prettier": "2.3.2",
     "typescript": "^3.7.5"
+  },
+  "resolutions": {
+    "glob-parent": "^6.0.1"
```

Comment:
Why are these changes needed?

Reply:
@rebecastilho I think it needs a rebase, then those unrelated changes won't appear in this PR